documentation/source/users/arkane/credits.rst

```diff
@@ -24,10 +24,10 @@ Developers: (rmg_dev@mit.edu)
 - Dr. D.S. Ranasinghe
 - Dr. R.J. Gillis
 - Dr. A.M. Payne
-- Asst. Prof. Y.-P. Li
-- X. Dong
-- K.A. Spiekermann
-- H. Wu
+- Prof. Y.-P. Li
+- Dr. X. Dong
+- Dr. K.A. Spiekermann
+- Mr. H. Wu
```

Comment:
Either remove Mr. here or add it for all the appropriate people in RMG credits (I prefer the former)